submodules/patternfly-core

```diff
@@ -0,0 +1 @@
+Subproject commit 1bcef08c25f021e58481847c805cba4886056b89
```

Comment:
I added the submodule with:
```
git submodule add https://github.com/patternfly/patternfly.git submodules/patternfly-core
```

then, within the submodule folder I did:
```
git checkout v3.7.0
```
and committed the change in the parent repo.  This way, all updates of the submodule will resolve to the same commit - the one with the commit hash corresponding to the v3.7.0 tag.